files/01_setup_disabling_of_inactive_freeipa_users.sh

```diff
@@ -6,11 +6,16 @@ set -o pipefail
 
 # Set up this instance to disable inactive FreeIPA users.
 function setup {
-  # Disable the "KDC:Disable Last Success" password plugin feature.
-  # We don't want this feature enabled because we want to be able to
-  # disable inactive users, which requires us to be able to determine
-  # the last time a user authenticated.
-  #
+  disable_password_plugin_feature
+  create_role_and_privilege
+  enable_systemd_timer
+}
```

Comment:
A small nitpick but I'd prefer this function declaration to be the last one since it uses the three other functions.